plugin/src/sbt-test/sbtix/simple/test

```diff
@@ -1,5 +1,10 @@
 > compile
 > genNix
-$ exists repo.nix
+> reload plugins
+> genNix
+$ exec mv repo.nix repo-build.nix
```

Comment:
> We could then have sbtix-merge and/or sbtix-gen-merge. One merges the files into a repo.nix. The other generates and merges in one go.

I'd rather just integrate the merging into sbtix-gen. But I think it's fine to keep `sbtix-gen-all` and co. until then.

> I find myself deleting repo.nix a lot. We should probably name it something we can add to .gitignore.

That... sounds like a bug. AFAIK, running `sbtix-gen` should generate an equivalent `repo.nix` regardless of whether one already existed. I'd rather not encourage `.gitignore`ing it, since it's relatively small, and useful for determinism.